src/main/java/com/artipie/rpm/meta/XmlPackage.java

```diff
@@ -45,23 +49,44 @@
             new MapEntry<>("", "http://linux.duke.edu/metadata/common"),
             new MapEntry<>("rpm", "http://linux.duke.edu/metadata/rpm")
         )
-    ),
+    ) {
+        @Override
+        public PackageOutput.FileOutput output() throws IOException {
+            return new PrimaryOutput(Files.createTempFile(this.tempPrefix(), XmlPackage.SFX));
+        }
+    },
 
     /**
-     * Metadata others.xml.
+     * Metadata other.xml.
      */
-    OTHERS(
+    OTHER(
         "otherdata",
         new MapOf<String, String>(new MapEntry<>("", "http://linux.duke.edu/metadata/other"))
-    ),
+    ) {
+        @Override
+        public PackageOutput.FileOutput output() throws IOException {
+            return new OthersOutput(Files.createTempFile(this.tempPrefix(), XmlPackage.SFX));
+        }
+    },
 
     /**
      * Metadata filelists.xml.
      */
     FILELISTS(
         "filelists",
         new MapOf<String, String>(new MapEntry<>("", "http://linux.duke.edu/metadata/filelists"))
-    );
+    ) {
+        @Override
+        public PackageOutput.FileOutput output() throws IOException {
+            return new FilelistsOutput(Files.createTempFile(this.tempPrefix(), XmlPackage.SFX));
+        }
+    };
+
+    /**
+     * File suffix.
+     * @checkstyle ConstantUsageCheck (5 lines)
+     */
+    private static final String SFX = ".xml";
```

Comment:
@olenagerasimova what's the problem with this checkstyle `ConstantUsageCheck` issue?  Also, can we use `SUFFIX` name?

Reply:
@g4s8 qulice is complaining 
```
Checkstyle: src/main/java/com/artipie/rpm/meta/XmlPackage.java[88]: Private constant "SUFFIX" is not used (ConstantUsageCheck)
```
seems like a bug, I'll check this one in qulice repo and create a bug of necessary